packages/@aws-cdk/aws-kinesisfirehose-destinations/lib/processor.ts

```diff
@@ -0,0 +1,116 @@
+import * as iam from '@aws-cdk/aws-iam';
```

Comment:
I wasn't sure the best place to put this actually. The reason I put it here is because it is only ever used in the `firehose-destinations` package. But, as a user, I guess it would make more sense to say something like `new firehose.LambdaProcessor()` instead of `new destinations.LambdaProcessor()`. Will move it.